ngraph/python/src/ngraph/ops/opset4/opset4.py

```diff
@@ -0,0 +1,111 @@
+# ******************************************************************************
+# Copyright 2017-2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+"""Factory functions for all ngraph ops."""
+from typing import Callable, Iterable, List, Optional, Set, Union
+
+import numpy as np
+
+from ngraph.impl import Node, Shape
+from ngraph.impl.op import Constant, GetOutputElement, Parameter
+from ngraph.utils.decorators import binary_op, nameable_op, unary_op
+from ngraph.utils.input_validation import (
+    assert_list_of_ints,
+    check_valid_attributes,
+    is_non_negative_value,
+    is_positive_value,
+)
+from ngraph.utils.node_factory import NodeFactory
+from ngraph.utils.tensor_iterator_types import (
+    GraphBody,
+    TensorIteratorSliceInputDesc,
+    TensorIteratorMergedInputDesc,
+    TensorIteratorInvariantInputDesc,
+    TensorIteratorBodyOutputDesc,
+    TensorIteratorConcatOutputDesc,
+)
+from ngraph.utils.types import (
+    NodeInput,
+    NumericData,
+    NumericType,
+    ScalarData,
+    TensorShape,
+    as_node,
+    as_nodes,
+    get_dtype,
+    get_element_type,
+    get_element_type_str,
+    make_constant_node,
+)
+
+
+def _get_node_factory(opset_version: Optional[str] = "opset2") -> NodeFactory:
+    """Return NodeFactory configured to create operators from specified opset version."""
+    if opset_version:
+        return NodeFactory(opset_version)
+    else:
+        return NodeFactory()
+
+
+# ------------------------ ops ---------------------------------------------------
+
+@nameable_op
+def get_output_element(data: NodeInput, index: int, name: Optional[str] = None) -> Node:
```

Comment:
In general we should think about removing `get_output_element` from the pythonAPI. 
Anyway, `opset4` is not the best place for this op.

Reply:
You're right. We can remove it.